.pre-commit-config.yaml

```diff
@@ -1,13 +1,17 @@
 repos:
 -   repo: https://github.com/ambv/black
-    rev: 18.6b4
+    rev: 18.9b0
     hooks:
     - id: black
       args: [--safe, --quiet]
       exclude: functional|input|test/extension|test/regrtest_data|test/data
       python_version: python3.6
 -   repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v1.2.3
+    rev: v2.1.0
     hooks:
     -   id: trailing-whitespace
     -   id: end-of-file-fixer
+-   repo: https://github.com/pre-commit/mirrors-isort
```

Comment:
@Pierre-Sassoulas We should probably add an `isort` step in `tox.ini`, followed by a CI check in `.travis.yml`. That way we can reject non-conforming code with CI, rather than noticing it after merging a PR that introduced violations.

Reply:
Resolving this since I started a new conversation in the tox.ini.